packages/react-dom/src/__tests__/ReactDOMFiberAsync-test.js

```diff
@@ -545,6 +549,68 @@ describe('ReactDOMFiberAsync', () => {
       // Therefore the form should have been submitted.
       expect(formSubmitted).toBe(true);
     });
+
+    // @gate enableUnknownLane
+    it.skip('batch default updates with existing unknown updates', () => {
```

Comment:
I'm not sure if these test represent the actual behavior we want (and I want to confirm before implementing this piece).